### ISSUE TITLE
program.out permissions

### ISSUE THREAD

**Issue by deprov447** [awknoman/ASCII-Face-Generator#2]
program.out isn't executable by default
`sudo chmod +x program.out`

**Comment by yasht01** [comment]
Yeah I could have used system() command to first change the read permissions, but I think it will also be system dependent because the file permissions in certain folders is set by the user or by system. Thanks for the help. I will change the code slightly.
